<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>818</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пол</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Sex_M">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>М</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton__Sex_W">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Ж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Str">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сила</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ловкость</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Int">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Интеллект</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Luck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удача</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Str">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Dex">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Int">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Luck">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Score_Left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Осталось очков</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Score_Left">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>510</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="Line" name="line_Hor">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>490</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_Vert">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>20</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Здоровье</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Def">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Защита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MP">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Мана</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Atk">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Атака</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP_Value">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Почти не осталось</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Def_Value">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>190</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Можно попробовать убежать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MP_Value">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Что это?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Atk_Value">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Главное не попасть в себя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Class">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Класс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Class_Value">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задохлик</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Race">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Раса</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Race">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Люди</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эльфы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дварфы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Орки</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>46</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
